crates/core/src/db/datastore/traits.rs

```diff
@@ -121,7 +123,6 @@ pub trait MutTxDatastore: TxDatastore + MutTx {
     fn schema_for_table_mut_tx<'tx>(&self, tx: &'tx Self::MutTx, table_id: TableId) -> Result<Cow<'tx, TableSchema>>;
     fn drop_table_mut_tx(&self, tx: &mut Self::MutTx, table_id: TableId) -> Result<()>;
     fn rename_table_mut_tx(&self, tx: &mut Self::MutTx, table_id: TableId, new_name: &str) -> Result<()>;
-    fn table_id_exists(&self, tx: &Self::MutTx, table_id: &TableId) -> bool;
```

Comment:
I guess this change was already merged into `master` somehow?

Reply:
no, this has been part of `TxId`, not required with `MutTxIxd`

Reply:
Well, I would assume you might still need to check to see if a table id exists in the context of a mutable transaction, no?

Reply:
Not for this case, method is only required during query compiling to `ast`, now whole compiling stage works with read-tx only.

Reply:
Okay, but for symmetry's sake it would make sense for this to be a thing you might want to do during a mutable transaction even if it's unused. I'm fine skipping it for now, and adding it later though.